ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/util/ScalablePushUtilTest.java

```diff
@@ -45,47 +44,39 @@
 @RunWith(MockitoJUnitRunner.class)
 public class ScalablePushUtilTest {
 
-  private static final Map<String, Object> overrides = ImmutableMap.of(
-      KsqlConfig.KSQL_QUERY_PUSH_V2_ENABLED, true,
```

Comment:
Why is this being removed?

Reply:
It wasn't being used anywhere AFAICT.

Reply:
Let's leave it in since conceptually it's important -- that feature flag enables scalable push queries. Otherwise this entire test file is kind of moot.

Reply:
This is in `expectIsSPQ()` and wasn't being used in any of Alan's tests- we're just passing it in through the main config, not the override. Given that, do you still think it's worth putting in? I'd want to understand the utility better before adding it.